app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -31,7 +30,7 @@ class MainActivity : ComponentActivity() {
                 // A surface container using the 'background' color from the theme
                 Surface(color = MaterialTheme.colors.background,
                 modifier = Modifier.fillMaxWidth()) {
-                    RecycleSearch("Android")
+                    DefaultPreview()
```

Comment:
Function names should be descriptive... I'm not sure that one of these is more descriptive than the other.